pkgs/data/fonts/libertine/default.nix

```diff
@@ -38,6 +41,9 @@ stdenv.mkDerivation {
     install -m444 -Dt $out/share/texmf/fonts/map       *.map
   '';
 
+  outputHashMode = "recursive";
+  outputHash = "sha256-/uFOwccWksnShoOhCnhvNFQFswmupUyFiUNkgF/JuUw=";
```

Comment:
Never seen this before in a normal derivation, what is the reason for adding the output hash?

Reply:
I discovered `outputHash` while looking at how fonts are usually packaged. Right now on master there are 18 fonts (over 86 which are using mkDerivation) that are using an output hash.
I believe it was done for fonts because the output content does not change, even if input dependencies change.

If you don't want a fixed outputHash, I can remove it!

Reply:
Please remove this again. There is no need to do this and it disabled the build sandbox.

Reply:
Fixed, sorry.